src/backend/distributed/operations/node_protocol.c

```diff
@@ -462,12 +464,14 @@ ResolveRelationId(text *relationName, bool missingOk)
 List *
```

Comment:
Similar to https://github.com/citusdata/citus/pull/6591/files#r1066806809
you should extend this function description as well.